WordPress/Classes/ViewRelated/Pages/BasePageListCell.h

```diff
@@ -1,19 +1,33 @@
 #import <UIKit/UIKit.h>
 #import "WPPostContentViewProvider.h"
 
+@class AbstractPost;
+@class BasePageListCell;
 
-@protocol BasePageListCellDelegate <NSObject>
-@optional
-- (void)cell:(nonnull UITableViewCell *)cell receivedMenuActionFromButton:(nonnull UIButton *)button forProvider:(nonnull id<WPPostContentViewProvider>)contentProvider;
-- (void)cell:(nonnull UITableViewCell *)cell receivedRestoreActionForProvider:(nonnull id<WPPostContentViewProvider>)contentProvider;
-@end
-
+/// A block that represents an action triggered by tapping on a button in a cell.
+///
+/// @param      cell        The cell that contains the button that was tapped.
+/// @param      button      The button that was tapped.
+/// @param      post        The post represented by the cell that was tapped.
+///
+typedef void(^BasePageListCellActionBlock)(BasePageListCell* _Nonnull cell,
```

Comment:
Rather nitpicky, but is there a reason that `*` is stuck to the type here, but stuck to the name elsewhere in this file?